pkg/types/gcp/validation/platform.go

```diff
@@ -24,19 +24,23 @@ var (
 		"asia-northeast2":         "Osaka, Japan",
 		"asia-northeast3":         "Seoul, South Korea",
 		"asia-south1":             "Mumbai, India",
+		"asia-south2":             "Delhi, India",
 		"asia-southeast1":         "Jurong West, Singapore",
 		"asia-southeast2":         "Jakarta, Indonesia",
 		"australia-southeast1":    "Sydney, Australia",
+		"australia-southeast2":    "Melbourne, Australia",
 		"europe-central2":         "Warsaw, Poland",
 		"europe-north1":           "Hamina, Finland",
 		"europe-west1":            "St. Ghislain, Belgium",
 		"europe-west2":            "London, England, UK",
 		"europe-west3":            "Frankfurt, Germany",
 		"europe-west4":            "Eemshaven, Netherlands",
 		"europe-west6":            "Zürich, Switzerland",
+		"europe-west8":            "Milan, Italy",
```

Comment:
Note that "europe-west8" Region doesn't support "N1" Machine Types too.

Reply:
@barbacbd  Let's not add `europe-west8` then for this reason. n1 is default, at least until #5815 merges. We are currently in the process of figuring out how to best handle adding new regions.